src/test/java/DilemmaDetector/Simulator/RigidBodyMapperTest.java

```diff
@@ -0,0 +1,59 @@
+package DilemmaDetector.Simulator;
+
+import generator.BaseScenarioGenerator2;
+import generator.Model;
+import generator.ObjectNamer;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.semanticweb.owlapi.apibinding.OWLManager;
+import org.semanticweb.owlapi.model.OWLOntology;
+import org.semanticweb.owlapi.model.OWLOntologyCreationException;
+import org.semanticweb.owlapi.model.OWLOntologyManager;
+import project.MyFactory;
+import project.Vehicle;
+
+import java.io.File;
+import java.lang.reflect.InvocationTargetException;
+import java.util.List;
+
+public class RigidBodyMapperTest {
+
+    public static final String baseIRI = "http://webprotege.stanford.edu/";
+    private Model model;
+
+    @Before
+    public void init() throws NoSuchMethodException, IllegalAccessException, InvocationTargetException, OWLOntologyCreationException {
+        OWLOntologyManager ontologyManager = OWLManager.createOWLOntologyManager();
+        OWLOntology ontology = ontologyManager.loadOntologyFromOntologyDocument(new File("src/main/resources/traffic_ontology.owl"));
+        MyFactory factory = new MyFactory(ontology);
+        BaseScenarioGenerator2 baseScenarioGenerator2 = new BaseScenarioGenerator2(factory, baseIRI);
+        this.model = baseScenarioGenerator2.generate();
```

Comment:
Tutaj będziesz potrzebował faktycznie model, ale trzeba pomyśleć, czy on musi być generowany przez baseScenarioGenerator? Najlepiej usunąć jakąkolwiek losowość z unit testów, one mają być deterministyczne, a z tego co mi wiadomo to generatory są losowe u nas. A nawet jak nie, to powinien być zrobiony mock Modelu, nie potrzebujesz scenariusza aby potestować Mapper.